library/ws-widget/collection/src/lib/card-content/card-content.component.scss

```diff
@@ -700,6 +700,13 @@ $title-line-count: 2;
       box-sizing: border-box;
       width: calc(100% - 315px);
 
+      @media (max-width:992px) {
```

Comment:
Dont use @media separtely, we already have the proper mixins for all the screens